cards/README.md

```diff
@@ -1,53 +1,54 @@
 
 <div class="info-box">
     <small>Important note</small><br/>
-    To use the <strong>Create Card</strong> endpoint you must use the test public key. This only applies to this endpoint. Using the <strong>Create Card</strong> endpoint on live environment requires your company to be PCI Compliant. If you want to use it instead of the SDKs, please contact us at developer@fidel.uk.
+    To use the <strong>Create Card</strong> endpoint, you must use the test public key. Using the <strong>Create Card</strong> API endpoint on live environment requires your company to be PCI Compliant (and will use the secret key. If you want to use it instead of the SDKs, please contact us at developer@fidel.uk.
 </div>
-
 ##### Create sample cards in test mode using the API Playground.
 
 ![Create card](https://docs.fidel.uk/assets/images/create-card.png "Create card")
 
-If the card is successfully linked, the newly created card object is returned in JSON and displayed in the response body box. You can also use the [Web SDK](/web-sdk) to create cards in test environment using your SDK test key. If an error occurs on card creation, you receive the error message in the HTTP response body.
+When the card is successfully linked, the newly created card object is returned in JSON and displayed in the response body box. If the card lining fails, the failure object will be delivered instead.  
+### Testing Card Linking with the Web SDK
+You can also use the [Web SDK](/web-sdk) to create cards in test environment using your public test key. If an error occurs on card creation, you receive the error message in the HTTP response body.
 
-##### JSON Card object
+##### JSON Card Response Object
 
 ```json
-fileName:card.json
+fileName:cardResponse.json
 {
-  "id": "e3fff00f-ab85-4a0a-b572-08b464ebf67a",
-  "accountId": "a30e933f-cde1-4ac1-9a5e-acd329497a48",
-  "programId": "e3fff00f-ab85-4a0a-b572-08b464ebf67a",
-  "metadata": {
-    "id": "this-is-the-metadata-id",
-    "customKey1": "customValue1",
-    "customKey2": "customValue2"
-  },
-  "provider": "mastercard",
-  "type": "master-card",
-  "lastNumbers": "3183",
-  "expMonth": 1,
-  "expYear": 2018,
-  "expDate": "2018-01-01T00:00:00.000Z",
-  "countryCode": "GBR",
-  "mapped": true,
-  "live": true,
-  "created": "2017-02-13T17:02:12.535Z",
-  "updated": "2017-02-13T17:17:02.833Z"
+    "items":[
```

Comment:
let's add the metadata as well, same metadata can be used as in the previous example

Reply:
readded